nixos/modules/misc/ids.nix

```diff
@@ -414,6 +415,7 @@
       btsync = 113;
       #minecraft = 114; # unused
       #monetdb = 115; # unused (not packaged), removed 2016-09-19
+      vault = 115;
```

Comment:
I found out, it is for "file" storage backend, the directory name is configurable and might be different than ```/mnt/lib/vault```.
So the logic is changed: https://github.com/NixOS/nixpkgs/pull/26907/commits/09dd5ec2dc79e3183803943e4a3947bbcf9ad2c8

Anyway, we need someone experienced with Vault and security in general to review. It is my first attempt to install Vault.